src/convertToObject.js

```diff
@@ -1,18 +1,18 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  sourceString
+    .split(';')
+    .filter(cssString => cssString.trim().length)
```

Comment:
if you got empty string after `cssString.trim()` `Undefined` will convert in false anyway. You don't need to get the length of a string so the `0 length` converts into false.